dotnet/src/Experimental/Agents/Extensions/AssistantsKernelFunctionExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Json.More;
+using Microsoft.SemanticKernel.Experimental.Agents.Models;
+
+namespace Microsoft.SemanticKernel.Experimental.Agents;
+
+internal static class AssistantsKernelFunctionExtensions
```

Comment:
Cosmetic - `AgentKernelFunctionExtensions` or maybe just `KernelFunctionExtensions`?

Reply:
This seems to be related to specific OpenAI Assistant stuff, and is internal only. Leaving it for now but we can change it if needed in a follow up.